### PR TITLE
Makes all lamdas pass through their inputs.

### PR DESCRIPTION
**Pull request by AWare**
To allow us to pass things to later lamdas, I'm passing through the input to output in all of the lamdas.
If we think this is a good move, then I'll fix the tests.

### CODE REVIEW

File: src/salesforce_uploader.js [hunk 1]
```diff
@@ -12,10 +12,17 @@ export function handler (input:?any, context:?any, callback:Function) {
     callback(new NamedError('inputError', 'Input did not contain filename'))
     return
   }
+
+  if (input.preventUpload !== null && input.preventUpload) {
+    callback(new NamedError('preventUploadError', 'Upload to salesforce disabled.'))
```
Comment (pvighi) [inline src/salesforce_uploader.js]: wouldn't this make the whole state machine execution fail?

Comment (pvighi) [inline src/salesforce_uploader.js]: maybe we could skip the last step using a choice state in the state machine:
http://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-choice-state.html

Comment (AWare) [inline src/salesforce_uploader.js]: Yes, would we want it to show as successful?

Comment (pvighi) [inline src/salesforce_uploader.js]: I think it would be better no never see failed executions in PROD unless it actually failed

**Comment by AWare** [comment]
Moved the uploader code into Salesforce, which now gives the URL of the object it uploads. Email address and final cloudformation should be a new PR.*

*My latest commit maybe should be one too.

**Pull request merged**